app/views/govuk_publishing_components/components/_big_number.html.erb

```diff
@@ -17,6 +17,18 @@
 
 %>
 <% if number %>
+  <% # We amend the capture here if it ends with a "+" to account for small symbols awkwardly sitting above the baseline in our typeface and amend them to sit at the baseline for the purpose of visual harmony %>
+  <% # Currently the only small symbol used with big numbers are pluses so they are is specifically targetted in the below code %>
+  <% if number.is_a? String
```

Comment:
Wouldn't `<% ... %>` with the closing percent angle bracket not still work?

Reply:
You are right. I was caught up in how my editor handles individual comment lines in `erb` files. I'll update this.